unity/Assets/Scripts/Logic/TileResolver.cs

```diff
@@ -19,7 +19,7 @@ public void ApplyBehaviour(ITile tile)
         public int CalculatePoints(ITile tile)
         {
             return tile.Nature.RelevantCoordinates(tile.Coordinate)
-                .Sum(coordinate => tile.Type.ValueOfRelationshipTo(_map.GetTile(coordinate).Type.Type));
+                .Sum(coordinate => _map.GetTile(coordinate) != null ? tile.Type.ValueOfRelationshipTo(_map.GetTile(coordinate).Type.Type) : 0);
```

Comment:
Ok, I see, how this didn't end up in NullPointers before - it was done after the voids were placed.

If you'd prefer `GetTile` didn't return null values, we can go back to the intended implemenation I mentioned earlier. Relationships to tiles of type None are implemented anyway.

Reply:
Nah, this works fine for now. I don't see the point of generating tiles that might not be needed at all